### ISSUE TITLE
MRE and gel-packs

### ISSUE THREAD

**Issue by aMadReason** [game-icons/icons#487]
Just occurred to me that an MRE (meal-ready-to-eat) could be a useful general consumable for modern/sci-fi style games.

An addition to that a gel/bottle consumable like those that athletes consume for glucose gel could also be quite cool.

Thanks!

**Comment by DarkZaitzev** [comment]
I made some as I saw this post, send them by mail.

I do not know if they work.
![mre svg](https://user-images.githubusercontent.com/28576660/36495598-66293a9e-16fb-11e8-878c-1825d338ceed.png)

**Comment by aMadReason** [comment]
Something like these in SVG format would be a great addition thanks @DarkZaitzev

**Comment by DarkZaitzev** [comment]
Already send the file to @Delapouite so I suppose they'll be in the site soon...

**Comment by Delapouite** [comment]
Yes, I received the mail. Thanks! I'll split them soon into icons.
